src/OperationResponse.php

```diff
@@ -297,7 +352,10 @@ public function getOperationsClient()
      */
     public function cancel()
     {
-        $this->operationsClient->cancelOperation($this->getName());
```

Comment:
The docblock here mentions a status of `\Google\Rpc\Code::CANCELLED` being set to the response of `getError` - will that hold true with DIREGAPIC? This question applies generally to all the RPCs and error codes.

Reply:
I updated the doc block to be more clear! PTAL!

Reply:
The docblock LGTM, thanks! 👍 I just wanted to double check, do the error codes of `\Google\Rpc\Code` map properly to what we get back from a DIREGAPIC? An example would be, if someone has written a snippet to check for a specific error code with a oneplatform API, will that work correctly with DIREGAPIC?

Reply:
No, they do not map to the same codes.